app/src/main/res/values-it/strings.xml

```diff
@@ -23,8 +23,8 @@
     <string name="past_transactions">Transazioni Recenti</string>
     <string name="or">OPPURE</string>
     <string name="reconnect">Riconnetti</string>
-    <string name="help">Help</string>
-    <string name="help_text">This application does remote control for your full node.&lt;br>&lt;br>In order to let your Android phone connect to your desktop, you need to run the desktop SilentDragon and sync fully.&lt;br>&lt;br>After your node is synced, go to &lt;b>Apps -> Connect Mobile App&lt;/b> to view the connection QR Code, which you can scan from this app for connect SilentDragon.</string>
+    <string name="help">Aiuto</string>
+    <string name="help_text">Questa applicazione controlla in remoto l\'intero nodo.&lt;br>&lt;br>Per consentire all\'utente di utilizzare questa applicazione, è necessario eseguire l\'applicazione SilentDragon su un computer e sincronizzarsi completamente.  &lt;br>&lt;br>Quindi vai su 'Applicazione -> Connessione all'applicazione mobile per visualizzare un codice QR, che puoi scansionare da questa applicazione per collegarti ad esso. </string>
```

Comment:
There are few unescaped apostrophes in that string.

Reply:
Indeed, I take care of it.